app/controllers/fitbit_profiles_controller.rb

```diff
@@ -22,7 +22,9 @@ def show
     
     #grab activity measures for graphs
     if @fitbit_profile.activities
-      @activity = FitbitActivity.find_all_by_fitbit_profile_id(@fitbit_profile.id,:order => "date_logged")
+      @activity = FitbitActivity
+        .where(fitbit_profile_id: @fitbit_profile.id)
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.